ghcide/src/Development/IDE/Import/FindImports.hs

```diff
@@ -123,11 +123,12 @@ locateModule env comp_info exts targetFor modName mbPkgName isSource = do
 #if MIN_VERSION_ghc(9,3,0)
     OtherPkg uid
       | Just dirs <- lookup uid import_paths
+          -> lookupLocal uid dirs
```

Comment:
:information_source: I've seen that in many cases stylish-haskell has trouble to parse parts of code, because they are so partial and don't seem to make sense to the parser. Duplication of parts of the code - in order to make the blocks more complete - helps the parser to understand what's going on.

Reply:
Thanks for the context!